integration/core/test_api.py

```diff
@@ -371,8 +372,10 @@ def test_upgrade_links(client):
 
 
 def test_template_icon(client):
-    url = 'http://localhost:8088/v1-catalog/templates/orig:nfs-server?image'
-    response = requests.get(url, headers=DEFAULT_HEADERS)
+
+    url = 'http://localhost:8088/v1-catalog/templates/orig:nfs-server' + \
+        '?image&projectId=%s' % (DEFAULT_ENV)
+    response = requests.get(url, headers=headers(''))
```

Comment:
fwiw, completely omitting the headers arg would have worked as well, but this is fine, no need to change